app/controllers/katello/api/v2/organizations_controller.rb

```diff
@@ -22,7 +22,7 @@ class Api::V2::OrganizationsController < ::Api::V2::OrganizationsController
         param :smart_proxy_ids, Array, N_("Smart proxy IDs"), :required => false
         param :compute_resource_ids, Array, N_("Compute resource IDs"), :required => false
         param :medium_ids, Array, N_("Medium IDs"), :required => false
-        param :config_template_ids, Array, N_("Provisioning template IDs"), :required => false # FIXME: deprecated
+        param :config_template_ids, Array, N_("Provisioning template IDs. Will be removed in Katello 4.1."), required: false, deprecated: true
```

Comment:
this can be just removed completely. It hasn't actually worked since 2019: https://github.com/theforeman/foreman/commit/9b2913f383b16b36009e64f3b988b65e6ac272b4#diff-914259019ce9daa4464c1df28cf4fb803d34182acecc24cdd450e762ae1d19e7L26